scripts/event-log-checker.ts

```diff
@@ -43,33 +49,57 @@ async function run() {
 
     const events: any = await (await fetch(`http://localhost:${PORT}/event-log`)).json();
 
-    test('Should log 2 events', () => {
-      assert.equal(
-        events.length,
-        2,
-        `Expected 2 events but received ${
-          events.length
-        } instead. The following events were logged: ${JSON.stringify(events)}`
-      );
-    });
+    if (definition.noBoot) {
```

Comment:
We should move this stuff into the tests themselves for maintainability? Not necessarily as part of this work.

Reply:
Which tests? I don't love this approach for sure, but I am not sure how to structure it differently.